Jenkinsfile

```diff
@@ -43,7 +43,7 @@ pipeline {
         }
         stage('compiling edx-app-android') {
             steps {
-                writeFile file: './OpenEdXMobile/edx.properties', text: 'edx.dir = \'../edx-mobile-config/stage/\''  
+                writeFile file: './OpenEdXMobile/edx.properties', text: 'edx.dir = \'../edx-mobile-config/prod/\''
```

Comment:
Could you describe why this used to mention the stage config and now mentions the prod config?

Reply:
it will download the Prod build instead of the stage, because,
Stage data is Testing data and Automation test cases checks the app as real users behaviour, and we cannot run our test cases on testing/changeable data, and it has limited access.